### PR TITLE
Added code and tests to allow for more than one column to be added to…

### LINKED ISSUE #53: save_enrichment_df_to_graph only allows one column to be added
The save_enrichment_df_to_graph method only allows one column to be added because it is missing a comma in between the set statements.

### LINKED ISSUE #54: Make limit optional on get_from_neo
Add the ability to make the limit optional on get_from_neo.  Leave the default limit in there and maybe add another parameter such as unlimited=true.

### PR DESCRIPTION
**Pull request by bechbd**
… save_enrichment_df_to_graph (Issue 53).  Added unlimited parameter to get_from_neo to allow for pulling an unlimited size of dataframe (Issue 54)

### CODE REVIEW

File: modules/Neo4jDataAccess.py [hunk 1]
```diff
@@ -206,17 +206,20 @@ def __get_neo4j_graph(self, role_type):
             self.graph = None
         return self.graph
 
-    def get_from_neo(self, cypher, limit=1000):
+    def get_from_neo(self, cypher: str, limit=1000, unlimited=False):
         graph = self.__get_neo4j_graph('reader')
         # If the limit isn't set in the traversal then add it
         if not re.search('LIMIT', cypher, re.IGNORECASE):
```
Comment (lmeyerov) [inline modules/Neo4jDataAccess.py]: We need an opt-out for not forcing adding LIMIT, like if someone calls APOC stuff

Comment (lmeyerov) [reply modules/Neo4jDataAccess.py]: ex:

```
if not (limit is None) and not re.search('LIMIT', cypher, re.IGNORECASE):
    cypher = cypher + " LIMIT " + str(limit)
```

**Review by lmeyerov — DISMISSED** [review]
lgtm. see comment on allowing `limit=None` to opt-out of editing the user query.

File: modules/Neo4jDataAccess.py (hunk 1 above)
Comment (bechbd) [reply modules/Neo4jDataAccess.py]: My bad.  now I see what you mean.  Let me fix that before we merge.

**Review by lmeyerov — APPROVED** [review]

**Pull request merged**